src/workflow/Files.py

```diff
@@ -0,0 +1,215 @@
+from pathlib import Path
+import string
+import random
+import shutil
+import streamlit as st
```

Comment:
can this be written without accessing the st. "workflow-dir" ?
Can't you just create a Files object with the current workflow-dir? or does this change in between?

Reply:
It does not change in between, but unlike the other classes (Logger, ParameterManager, CommandExecutor) new individual Files objects need to be created while setting up the workflow. Of course, it would be possible to pass the workflow-dir every time. To avoid this repetition, I decided to get the workflow-dir from session_state.